src/app.js

```diff
@@ -26,52 +26,55 @@ function main(sources) {
     time: new Date()
   }]);
 
+  const intent = DOM => {
+    const changeMessage$ = sources.DOM
+      .select('.input-message')
+      .events('input')
+      .map(e => e.target.value)
+      .share();
```

Comment:
That's useful, thank you. One thing I don't understand though; why do we need this here?